custom_components/frigate/__init__.py

```diff
@@ -405,6 +405,10 @@ def update_unique_id(entity_entry: er.RegistryEntry) -> dict[str, str] | None:
                     "sensor_object_count",
                     m.group("cam_obj"),
                 ],
+                re.compile(rf"^{DOMAIN}_(?P<cam_obj>\S+)$"): lambda m: [
```

Comment:
I don't think you need to add this, as these sensors will never exist in a `v1` config flow (and so would not need to be updated).

Reply:
Ah I see. Wasn't exactly sure how this part worked. Removed!